lib/src/facebook_login.dart

```diff
@@ -157,6 +158,14 @@ class FacebookLogin {
     return FacebookLoginResult.fromMap(loginResultData.cast<String, dynamic>());
   }
 
+  Future<FacebookLoginResult> expressLogin() async {
+    if (debug) _log('Trying to expressLogin');
```

Comment:
I suggest to add `assert` here instead of making fake iOS implementation
```suggestion
    assert(Platform.isAndroid);
    if (debug) _log('Trying to expressLogin');
```